src/components/definition.tsx

```diff
@@ -46,14 +51,32 @@ export default class Definition extends React.Component<IProps, IState> {
           imageVisible &&
           <div className={css.imageContainer}>
             <img src={imageUrl} />
-            <div className={css.caption}>{imageCaption}</div>
+            {
+              imageCaption &&
+              <div className={css.caption}>
+                {imageCaption}
+                {
+                  textToSpeechAvailable() &&
+                  <span className={icons.iconButton + " " + icons.iconAudio} onClick={this.readImageCaption}/>
+                }
```

Comment:
Seems good to pull this out into a render method that takes the `this.read*Caption` as a parameter